LICENSE.txt

```diff
@@ -1,27 +0,0 @@
-Copyright (c) 2015-2018, conda-forge
```

Comment:
Do not delete this file. It's the top level license for conda-forge

Reply:
sorry about that!